### PR TITLE
Add comment to explain schema_url fields

### PR DESCRIPTION
**Pull request by tigrannajaryan**
Resolves https://github.com/open-telemetry/opentelemetry-proto/issues/502

### CODE REVIEW

**Review by MadVikingGod — APPROVED** [review]

**Review by MrAlias — APPROVED** [review]

File: opentelemetry/proto/logs/v1/logs.proto [hunk 1]
```diff
@@ -55,6 +55,9 @@ message ResourceLogs {
   // A list of ScopeLogs that originate from a resource.
   repeated ScopeLogs scope_logs = 2;
 
+  // The Schema URL, if known. This is the identifier of the Schema that the resource data
+  // is recorded in. To learn more about Schema URL see
+  // https://github.com/open-telemetry/opentelemetry-specification/tree/main/specification/schemas#schema-url
```
Comment (jsuereth) [inline opentelemetry/proto/logs/v1/logs.proto:60]: Could we point this at the opentelemetry.io website?

I think we have more control over redirects there in the event we refactor the specification.

**Review by jsuereth — APPROVED** [review]
One not-blocking comment.

**Review by carlosalberto — APPROVED** [review]

File: opentelemetry/proto/logs/v1/logs.proto (hunk 1 above)
Comment (tigrannajaryan) [reply opentelemetry/proto/logs/v1/logs.proto:60]: I can't find the schema pages anywhere at https://opentelemetry.io/docs/specs/otel/

Looks like they are not published.

Comment (tigrannajaryan) [reply opentelemetry/proto/logs/v1/logs.proto:60]: Nevermind, it is there https://opentelemetry.io/docs/specs/otel/schemas/#schema-url

Comment (tigrannajaryan) [reply opentelemetry/proto/logs/v1/logs.proto]: Links now point to opentelemetry.io website.

**Pull request merged**